src/com/lilithsthrone/game/Game.java

```diff
@@ -3573,11 +3494,6 @@ public void setResponsePointer(int responsePointer) {
 	public boolean isPlayerTileFull() {
 		return getActiveWorld().getCell(getPlayer().getLocation()).getInventory().isInventoryFull();
 	}
-
-
-	public boolean isDebugMode() {
-		return Main.getProperties().hasValue(PropertyValue.debugMode);
-	}
```

Comment:
What's the purpose of this change?

Reply:
It's the result of a bad merge. Thanks for the note, I'll clean it up.